aptos-move/framework/move-stdlib/sources/hash.move

```diff
@@ -3,12 +3,20 @@
 /// The functions in this module are natively declared both in the Move runtime
 /// as in the Move prover's prelude.
 module std::hash {
+    use std::bcs;
+
     // TODO: move sip_hash into aptos_framework
-    native public fun sip_hash<MoveValue>(v: &MoveValue): u64;
-    native public fun sha2_256(data: vector<u8>): vector<u8>;
-    native public fun sha3_256(data: vector<u8>): vector<u8>;
+    public fun sip_hash<MoveValue>(v: &MoveValue): u64 {
```

Comment:
How about naming it something like `sip_hash_from_value`?

Reply:
I don't feel strongly either way. @zekun000?

Reply:
Okay! Renamed!